backend/domain/transaction/transaction_crud.py

```diff
@@ -0,0 +1,53 @@
+from sqlalchemy.orm import Session
+from domain.account.account_schema import (
+    AccountCreate,
+    AccountUpdate,
+)
+from domain.transaction.transaction_schema import(
+    TransactionCreate,
+    TransactionUpdate,
+)
+from domain.account.account_crud import(
+    get_account_by_id,
+)
+from models import User, Account, Transaction
+import uuid
+from datetime import datetime, timedelta, timezone
+from starlette import status
+from fastapi import HTTPException
+
+def create_transaction(
+        db: Session,
+        transaction_create: TransactionCreate,
+        account_id: str
+) -> Transaction:
+    """
+    Create new transaction
+    """
+    account = get_account_by_id(db, account_id)
+    db_transaction = Transaction(
+        id=str(uuid.uuid4()),
+        transaction_date=transaction_create.transaction_date,
+        transaction_amount=transaction_create.transaction_amount,
+        account_id=account
+    )
+
+    account.current_balance += db_transaction.transaction_amount
```

Comment:
We are updating the current_balance of the account here but are not saving it back to the database. I would create account_balance_update() in account_crud.py where it takes the account object and the new balance, and update the account balance.